.github/workflows/registry-consistency-ci.yml

```diff
@@ -12,7 +12,7 @@ env:
 permissions:
   contents: read
 jobs:
-  CI:
+  check:
```

Comment:
We could pick something else (instead of `check`) if we want. We could do e.g. `consistency`.